### PR TITLE
Add remove bookmark

### PR DESCRIPTION
**Pull request by barbayjuliette**
You can now bookmark trails from index or show page. Icon adapts.
Right now, page still refreshes every time, ajax doesn't work. Will work on that afterward. 
I added the distance to the card and changed their position. They wrap only between breakpoints.
Add margin under the cards-container. 
I used a gem for the bookmarks: acts_as_favoritor, so the bookmark table, controllers,... can be deleted.
Will do that in next branch too, and I will then also need to update the show page for the bookmarks.

### CODE REVIEW

**Review by lxingern — DISMISSED** [review]
Seems to work fine, ok to merge.

Some things/suggestions to work on:
- Creating the bookmarks index page
- Maybe we can make the bookmark icon yellow, if active? Instead of switching to white (for index) or black (for show page)

**Comment by barbayjuliette** [comment]
I changed the color of the bookmark to trailyellow, and also of the stars

**Comment by barbayjuliette** [comment]
- I added the index page for the bookmarks with the link on the navbar 2
- I deleted the bookmark model and table (For the bookmarks, we are using the act_as_favorite gem, with the favorite model /table)

**Comment by barbayjuliette** [comment]
I fixed the problems after the merge.
There is only one thing to fix:
- The cards in the trip index are not correctly styled. @lxingern for your PR, use the latest version of the cards that is in the partial.

File: app/views/trails/show.html.erb
Comment (lxingern) [inline app/views/trails/show.html.erb:83]: Can you delete this line (line 83)? Currently the Create Trip button is stretching across the page and it's because of this line.

image

**Review by lxingern — APPROVED** [review]
Ok to merge after removing the line I mentioned in my previous comment. 

Something else to work on when we have time - currently the entire page is still refreshing whenever a bookmark is created/deleted. Possible to ajaxify this?

**Review by lxingern — APPROVED** [review]
Ok, thanks!

**Pull request merged**
